in_text/text_03_config.yml

```
@@ -139,7 +139,7 @@ entities:
     -
       attr-label: A
       attr-def: >-
-        Comma-separated list of lake areas
+        Comma-separated list of lake areas 
```

lake areas for which lakes? is there a companion to this that explains what these represent?
```suggestion
        Comma-separated list of lake areas
```

---

Documented this way last year this way. https://github.com/USGS-R/drb-temp-data-release/blob/5473cfbf811b4d22f9df3f0399ed55898ac92f64/in_text/text_03_config.yml#L132-L139. 

Updated for clarity sake here Updated via this commit https://github.com/USGS-R/forecast-data-release-v2/pull/13/commits/1330ea127ec2c677d95b1cdebcc66f20f54b3190